### PR TITLE
(feat) (O3-2573): Implement the new snackbar patient management (patient-registration-app)

### PR DESCRIPTION
**Pull request by usamaidrsk**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
- Implement new snackbar in patient-registration-app, to replace the toast

## Screenshots

image
image
image

## Related Issue

[O3-2573](https://openmrs.atlassian.net/browse/O3-2573)

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-registration-app/src/patient-registration/field/field.resource.ts [hunk 1]
```diff
@@ -25,10 +26,11 @@ export function useConceptAnswers(conceptUuid: string): { data: Array<ConceptAns
     openmrsFetch,
   );
   if (error) {
-    showToast({
+    showSnackbar({
       title: error.name,
-      description: error.message,
+      subtitle: error.message,
       kind: 'error',
+      isLowContrast: true,
```
Comment (hadijahkyampeire) [inline packages/esm-patient-registration-app/src/patient-registration/field/field.resource.ts:33]: This should be false, FYI all errors take false.

File: packages/esm-patient-registration-app/translations/es.json [hunk 2]
```diff
@@ -64,8 +63,9 @@
   "patientNameKnown": "¿Se sabe el nombre del paciente?",
   "patientRegistrationBreadcrumb": "Patient Registration",
   "registerPatient": "Registrar paciente",
-  "registrationSuccessToastDescription": "Ahora se puede encontrar al paciente buscándolo por su nombre o número de identificación",
-  "registrationSuccessToastTitle": "Nuevo paciente creado",
+  "registerPatientSuccessSnackbarDescription": "The patient can now be found by searching for them using their name or ID number",
```
Comment (hadijahkyampeire) [inline packages/esm-patient-registration-app/translations/es.json:66]: And here, let's change it to registerPatientSuccessSnackbarSubtitle

**Review by hadijahkyampeire — COMMENTED** [review]
Thanks @usamaidrsk just a few improvements needed here.

**Comment by hadijahkyampeire** [comment]
@usamaidrsk you need to run the command to extract translations so that the keys get updated. I think it is `yarn turbo extract translations`. As you can see the keys in the translation files did not get updated.

**Comment by usamaidrsk** [comment]
> yarn turbo extract translations

okay, I've done so. Thanks

**Review by hadijahkyampeire — APPROVED** [review]
LGTM thanks @usamaidrsk

**Pull request merged**